helm/alfresco-content-services/tests/deployment-repository_test.yaml

```diff
@@ -195,3 +195,59 @@ tests:
               cpu: "4"
               memory: "3000Mi"
         template: deployment-repository.yaml
+
+  - it: should have wait-db-ready initcontainer with external database
+    values: *testvalues
+    set:
+      database.external: false
+    asserts:
+      - equal:
+          path: spec.template.spec.initContainers[0].name
+          value: wait-db-ready
+        template: deployment-repository.yaml
+
+  - it: should not have wait-db-ready initcontainer with external database
+    values: *testvalues
+    set:
+      database.external: true
+    asserts:
+      - isEmpty:
+          path: spec.template.spec.initContainers
+        template: deployment-repository.yaml
+      - isNotEmpty:
+          path: spec.template.spec.volumes
+        template: deployment-repository.yaml
```

Comment:
Looks unrelated to the test or am I missing something?

Reply:
it is to check explicitly for an effect that the bug was causing (empty volumes)